docs/media-volume-range.md

```diff
@@ -1,18 +0,0 @@
-# `<media-volume-range/>`
```

Comment:
question(non-blocking): Any idea why did this not get tracked as a move?

Reply:
hmm not sure, will check this out